data_service/src/match_processing/match_parser.rs

```diff
@@ -0,0 +1,98 @@
+use riven::models::match_v5::Match;
+
+use crate::types::PuuidToChampionMapping;
+
+pub async fn parse_match(match_data: Match) -> Vec<PuuidToChampionMapping> {
+    // Get match data.
```

Comment:
nit: get rid of this comment now that the implementation is done

Reply:
resolved, please check